apps/src/music/blockly/blocks/samples.js

```diff
@@ -17,20 +15,16 @@ const isBlockInsideWhenRun = ctx => {
 export const playSound = {
   definition: {
     type: BlockTypes.PLAY_SOUND,
-    message0: '%1 play %2 at measure %3',
+    message0: 'play %1 at measure %2',
     args0: [
       {
-        type: 'field_image',
-        src: getStaticFilePath(PLAY_ICON),
-        width: 18,
-        height: 23,
-        alt: '*',
-        flipRtl: false,
-        name: 'image'
-      },
-      {
-        type: 'input_dummy',
-        name: 'sound'
+        type: 'field_sounds',
+        name: 'sound',
+        getLibrary: () => window.library,
+        playPreview: (id, onStop) => {
+          window.playPreview(id, onStop);
+        },
+        currentValue: 'pop/cafe_beat'
```

Comment:
Maybe a question for @moneppo but I wonder if we can have a default value here that makes it super obvious that you can drop down and select any sample. Like "no sample selected" or something?

Reply:
It's a good question.  One reason I added this default was so that you get a working sound as soon as you use the block.  I certainly think we should try to find an affordance (even just the down arrow, as discussed separately, or something more explicit) and/or add some teaching to help the user find the sound selector.